app/src/main/res/layout/fragment_home.xml

```diff
@@ -8,14 +8,54 @@
     android:gravity="center"
     android:orientation="vertical">
 
+    <ImageView
+        android:id="@+id/vWoloxImage"
+        android:layout_width="74dp"
```

Comment:
Remember to add all the "hardcoded" dimens to resources and to run a `Reformat code` to order the code